boards/arm/olimexino_stm32/olimexino_stm32_defconfig

```diff
@@ -20,6 +20,15 @@ CONFIG_UART_CONSOLE=y
 CONFIG_I2C=y
 CONFIG_I2C_STM32_V1=y
 
+# enable SPI
+CONFIG_SPI=y
+CONFIG_SPI_STM32=y
+CONFIG_SPI_STM32_INTERRUPT=y
+CONFIG_SPI_1=y
+CONFIG_SPI_1_IRQ_PRI=10
+CONFIG_POLL=y
```

Comment:
How is that SPI related? If the interrupt driver requires CONFIG_POLL=y, maybe it should select it.

Reply:
I just copy/paste some CONFIG_* from the spi_loopback prj.conf and forget to remove it. I will fix it.